Day2ConsoleAppHomework/Program.cs

```diff
@@ -1,15 +1,44 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Day2ConsoleAppHomework
 {
+    enum TransactionType { Buy, Sell }
+
     class Program
     {
+        string currency = "DKK";
+
         static void Main(string[] args)
         {
+            string currency = "DKK";
+
+            Console.WriteLine("Input the transaction type:");
+            var userInput = Console.ReadLine();
+            var transactionType = (TransactionType)Enum.Parse(typeof(TransactionType), userInput, true);
+
+            Console.WriteLine("Input the nominal of the trade:");
+            userInput = Console.ReadLine();
+            var nominal = float.Parse(userInput);
+
+            Console.WriteLine("Input the trade price:");
+            userInput = Console.ReadLine();
+            var tradePrice = float.Parse(userInput);
+            
+            Console.WriteLine("Input the trade price for the original trade (only used for sales):");
+            userInput = Console.ReadLine();
+            var origPrice = float.Parse(userInput);
+            
+            var sign=transactionType==TransactionType.Sell?-1:1;
+            var costValue = sign*nominal*tradePrice;
+
+            var factor = transactionType == TransactionType.Sell ? 1 : 0;
+            var prl = factor * (tradePrice - origPrice) * nominal;
+
+            var msg1 = "The calculated cost value is: {0:N2} {1}";
```

Comment:
please make a meaningfull names of the variables or get rid of them if the do not have any reason